src/components/fullHeight/index.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Box } from '@chakra-ui/react';
+
+import FullHeight from './index.tsx';
+
+<Meta title="Layout/FullHeight" component={FullHeight} />
+
+export const Template = () => {
+  return (
+    <FullHeight>
+      <Box backgroundColor="brand.primary" minHeight="100vh">
```

Comment:
It's a bit tricky to showcase the height of the container when not forcing the child with not enough content to take the same height...